code/modules/reagents/holder/holder.dm

```diff
@@ -268,9 +268,13 @@
 //If for some reason touch effects are bypassed (e.g. injecting stuff directly into a reagent container or person),
 //call the appropriate trans_to_*() proc.
 /datum/reagents/proc/trans_to(var/atom/target, var/amount = 1, var/multiplier = 1, var/copy = 0)
-	if(ismob(target))  //CHOMPEdit
-		return splash_mob(target, amount * multiplier, copy) //Touch effects handled by splash_mob
+	//CHOMPEdit Start, we only can do the splashing first on carbon (Runtime reasons)
+	if(iscarbon(target))
+		return splash_mob(target, amount * multiplier, copy)
+	//CHOMPEdit End
 	touch(target, amount * multiplier) //First, handle mere touch effects
+	if(ismob(target))

```

Comment:
Chompedit comments here and 277 (one below)

Reply:
no, reverted to upstream code.